label_studio/data_manager/managers.py

```diff
@@ -509,46 +510,54 @@ def __init__(self, expression, distinct=False, output_field=None, **extra):
         super().__init__(expression, distinct='DISTINCT ' if distinct else '', output_field=output_field, **extra)
 
 
-def annotate_completed_at(queryset):
-    if flag_set('fflag_feat_optic_161_project_settings_for_low_agreement_threshold_score_short', user='auto'):
-        return annotated_completed_at_considering_agreement_threshold(queryset)
-
+def newest_annotation_subquery():
     from tasks.models import Annotation
 
-    newest = Annotation.objects.filter(task=OuterRef('pk')).order_by('-id')[:1]
-    return queryset.annotate(completed_at=Case(When(is_labeled=True, then=Subquery(newest.values('created_at')))))
+    return Annotation.objects.filter(task=OuterRef('pk')).order_by('-id')[:1]
```

Comment:
Can we add a type signature here (and to the other functions in this file)? It's a bit confusing to me that this doesn't actually return a `Subquery` (and really, it looks like it could)?